CONTRIBUTING.md

```diff
@@ -16,23 +16,18 @@ To setup your environment:
   ```
 1. Install Pyenv versions for the Tox Suite
   ```bash
-  pyenv install 3.5.8
   pyenv install 3.6.10
   pyenv install 3.7.7
   pyenv install 3.8.2
   pyenv install pypy3.6-7.3.1
   ```
 1. Install Poetry
   ```bash
-  curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+  pip install poetry
   ```
-1. Install Tox
+1. Install Dependencies
   ```bash
-  pip install tox
-  ```
-1. Setup the local project versions (one for each env in the `envlist`)
-  ```bash
-  pyenv local 3.5.8 3.6.10 3.7.7 3.8.2 pypy3.6-7.1.1
+  poetry install -v
```

Comment:
i just installed with `poetry` then ran tox with `poetry run tox` (as below) and it seemed to handle everything

Reply:
since we are running tox in travis, maybe we don't need to test all versions locally

Reply:
ok, yea as i was actually developing i just did `poetry run pytest <my_test_file>` so i can update to that